Common++/header/GeneralUtils.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
-#include <string>
 #include <stdint.h>
+#include <string>
```

Comment:
I added that option later, but I think it's better to reorder as long as there is no build error?
I made the reordering to off because it may break the rest of the codebase.

Reply:
Once we agree on all the formatting rules, I suggest we re-run the existing Common++ codebase with `clang-format` to make sure the changes match the formatting rules

Reply:
I reset it.